src/ui-hlp/menu.cpp

```diff
@@ -375,7 +375,7 @@ static void uih_render(struct uih_context *c, dialogparam *d)
 static menudialog *uih_getcolordialog(struct uih_context *c)
 {
     if (c != NULL) {
-        uih_colordialog[0].defint = c->color;
+        uih_colordialog[0].definite = c->color;
```

Comment:
Correcting variable names is difficult, when not knowing where they emerge from.
I guess "defint" = "default value, integer"